packages/amplify-console-integration-tests/src/consoleHosting/utils.ts

```diff
@@ -34,3 +36,12 @@ export function loadAppIdFromTeamProviderInfo(cwd: string, currEnv: string) {
 function readJsonFile(jsonFilePath: string, encoding = 'utf8') {
   return JSON.parse(fs.readFileSync(jsonFilePath, encoding));
 }
+
+export function createTestProject(): string {
+  const projectName = `amplify-integ-${Math.round(Math.random() * 100)}-test-${Math.round(Math.random() * 1000)}`;
```

Comment:
we should move the similar function we have in amplify-e2e-tests to amplify-e2e-core and reuse it instead of creating a new one here

Reply:
the name should be changed to indicate it is a console test.